jarviscli/plugins/caesar_cipher.py

```diff
@@ -0,0 +1,84 @@
+from plugin import plugin
+from colorama import Fore
+
+
+@plugin("caesar cipher")
+def caesar_cipher_converter(jarvis, str):
+    option = get_option(jarvis)
+    if option == 1:
+        plain_to_cipher(jarvis)
+    elif option == 2:
+        cipher_to_plain(jarvis)
+    else:
+        return
+
+
+def get_option(jarvis):
+    jarvis.say("~> What can I do for you?", Fore.RED)
+    print("1: Convert plain text to cipher")
+    print("2: Convert cipher to plain text")
+    print("3: Exit")
+    print()
+
+    while True:
+        try:
+            option = int(jarvis.input("Enter your choice: ", Fore.GREEN))
+            if option == 3:
+                return
+            elif option == 1 or option == 2:
+                return option
+            else:
+                jarvis.say(
+                    "Invalid input! Enter a number from the choices provided.", Fore.YELLOW)
+        except ValueError:
+            jarvis.say(
+                "Invalid input! Enter a number from the choices provided.", Fore.YELLOW)
+        print()
+
+
+def plain_to_cipher(jarvis):
+    user_input = get_user_input(jarvis)
+    converted = ""
+
+    for i in user_input:
+        if i.isalpha():
+            if i.isupper():
+                converted += chr((ord(i) - 68) % 26 + 65)
+            else:
+                converted += chr((ord(i) - 100) % 26 + 97)
+        else:
+            converted += i
+
+    jarvis.say(converted, Fore.YELLOW)
+
+
+def cipher_to_plain(jarvis):
+    user_input = get_user_input(jarvis)
+    converted = ""
+
+    for i in user_input:
+        if i.isalpha():
```

Comment:
Good work!

Only noticed, you did not consider Unicode. Something like for example our German: ä, ü, ö, ß ;). As a suggestion, you could instead isalpha do a check similar to:
https://stackoverflow.com/questions/196345/how-to-check-if-a-string-in-python-is-in-ascii#answer-196392

Reply:
Ok, I'll fix it